README.md

```diff
@@ -4,22 +4,23 @@ Welcome to `phenix`!
 
 ## Building
 
-To build locally, you will need Golang v1.18, Node v14.2, Yarn 1.22, and Protoc
-3.14 installed. Once installed (if not already), simply run `make bin/phenix`.
+### Local build
+To build locally, you will need Golang v1.18, Node v14.2, Yarn 1.22, and Protoc 3.14 installed.
 
-If you don't want to install Golang and/or Node locally, you can also use Docker
-to build phenix (assuming you have Docker installed). Simply run
-`./docker-build.sh` and once built, the phenix binary will be available at
-`bin/phenix`. See `./docker-build.sh -h` for usage details.
```

Comment:
This got moved to `hack/build/docker-build.sh` just in case you want to leave this in the README.

Reply:
Totally missed that this got moved and not deleted. Updated README to point to it.